### DISCUSSION TITLE
Puan

### DISCUSSION THREAD

**Discussion by metehanguler34** [IBAX-io/go-ibax#1388] [Soru Cevap - Türkçe]
Telegram discord youtube join yaptık events kismindan oduller ne zaman nasıl gelecek

**Comment by Yusufknk** [comment]
Arkadaşlar bunun için yöneticilerle görüşüyoruz. İlk başta form olarak yapacaklardı ama birden fazla bot hesap ihtimali olduğu için bu yöntemi elediler.
